technical-resources/loans/loans.md

```diff
@@ -0,0 +1,40 @@
+In the protocol, we have two classifications of loans: open-term and fixed-term. Despite their notable distinctions, their underlying usage and functionality are fundamentally the same. They can be summarized as follows:
+
+1. Execution of loan financing.
+2. Withdrawal of funded amounts.
+3. Management of collateral, when necessary.
+4. Computation of payment quantities and timetables (capable of managing both amortized and interest-only payment formats).
+5. Conducting repossessions of collateral and leftover amounts in the event of a loan default.
+6. Claim of interest and principal from loans.
+7. Carrying out refinancing activities when a lender and borrower consent to new terms.
+8. Change of loan logic via upgradeability models.
+
+
+## Deployments
+Loans operate as proxy contracts that link to a particular implementation. Borrowers interested in establishing loans within the Maple ecosystem must do so through a Maple Loan Factory contract (FixedTermLoanFactory or OpenTermLoanFactory). These contracts store details about the current version of loans and potential upgrade paths.
+
+## Loan Creation Pre-Requisites
+
+The following pre-requisites must be met before the creation of a Loan.
+
+1. The Borrower must be set as valid in the MapleGlobals allowlist.
+2. The asset used for the funding of the Loan must be set as valid by the Governor in MapleGlobals.
+3. The asset used as collateral of the Loan must be set as valid by the Governor in MapleGlobals.
+4. The Lender must be a valid version of the respective type of [`Loan Manager`](../../loan-managers)
+
+
+## Key Differences Between Open-Term/Fixed-Term Loans
+- Open-Tem do not have a due date for which the principal and outstanding interest need to be paid back by.
+- Fixed-Term Loans may be required to have the borrower post Collateral whereas Open-Term Loans have no concept of Collateral.
+- Fixed-Term Loans may be amortizing or interest-only whereas Open-Term Loans are interest Only
+- Fixed-Term Loans require the full interest/principal(in case of amortizing) when making a payment regardless if the payment is made before the due date whereas Open-Term Loans prorate the interest payment based on the timestamp the payment is made.
```

Comment:
```suggestion
- Fixed-Term Loans require the full interest/principal (in case of amortizing) when making a payment regardless if the payment is made before the due date whereas Open-Term Loans prorate the interest payment based on the timestamp the payment is made.
```